_pages/activities.html

```diff
@@ -14,10 +14,10 @@ <h4>Learning Continues Outside of the Classroom</h4>
     <h4>English Now! School Calendar: Looking Ahead</h4>
     <p>Please see below for our school calendar. This calendar always shows our Friday Outings for the next one to two months, Special Events for the year, upcoming Community Service Weeks, and a listing of all school holidays. We hope we can welcome you to some of these activities!</p>
     <iframe src="https://calendar.google.com/calendar/embed?src=school.calendar%40english-now.com&ctz=America%2FNew_York" style="border: 0" width="100%" height="600" frameborder="0" scrolling="yes"></iframe>
+    <img src="/images/activities/npr-visit.jpg" style="display: block; float: left; margin-left: 0px; max-width: 500px; margin-top: 10px; margin-right: 20px;">
```

Comment:
I know you're just moving this from elsewhere, but you should probably make a CSS class for images like this. Inline styling should pretty much never be used.

Reply:
Sorry—I know you have told me not to use inline styling in the past. I wrongly assumed that inline was the convention for this website since the images on the page used it. I'll make a class and convert all other images on this page. Is this the sort of thing I should update globally or should images be converted when their pages are edited over time?

Reply:
Additionally, should the default inline CSS be removed and converted to a class on `<iframe>` elements like the Google Calendar on this page?

Reply:
No worries! You can leave the `iframe` CSS, I assume Google knows what they're doing.